### ISSUE TITLE
Maven Repository - Reactive Logstash Logging Spring Boot Starter

### ISSUE THREAD

**Issue by arkadiuszSzast** [piomin/spring-boot-logging#16]
Version published on the Maven Repository is probably outdated. I checked it and there is no option to configure it using` logging.logstash` properties. I guess the published version is: 15cfe1d3aa8b507e7b9f9e1847ea88648d5c7355. Can you check it?

**Comment by choweiyuan** [comment]
Hi I notice the same issue for [1.0.0.RELEASE on mvnrepository](https://mvnrepository.com/artifact/com.github.piomin/reactive-logstash-logging-spring-boot-starter/1.0.0.RELEASE), which isn't the same as [master](https://github.com/piomin/spring-boot-logging/blob/master/reactive-logstash-logging-spring-boot-starter/src/main/java/pl/piomin/logging/reactive/config/ReactiveSpringLoggingAutoConfiguration.java)

```
package pl.piomin.logging.reactive.config;

import org.springframework.beans.factory.annotation.Value;
import pl.piomin.logging.reactive.filter.ReactiveSpringLoggingFilter;

import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Configuration;
import pl.piomin.logging.reactive.util.UniqueIDGenerator;

@Configuration
public class ReactiveSpringLoggingAutoConfiguration {

	private static final String LOGSTASH_APPENDER_NAME = "LOGSTASH";

	private String url = "localhost:8500";
	private String ignorePatterns;
	private boolean logHeaders;
	private boolean useContentLength;
	private String trustStoreLocation;
	private String trustStorePassword;
	@Value("${spring.application.name:-}")
	String name;

	@Bean
	public UniqueIDGenerator generator() {
		return new UniqueIDGenerator();
	}

	@Bean
	public ReactiveSpringLoggingFilter reactiveSpringLoggingFilter() {
		return new ReactiveSpringLoggingFilter(generator(), ignorePatterns, logHeaders, useContentLength);
	}

}
```

**Comment by choweiyuan** [comment]
@piomin any update on this (to publish reactive version to maven central?)

**Comment by arkadiuszSzast** [comment]
can you provide any update? @piomin

**Comment by piomin** [comment]
Hi. Yes. I have just published version 1.3.0 on Maven Central. Could you verify that it solves your errors?

**Comment by choweiyuan** [comment]
Looks good to me @piomin , thanks for that
